1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,7 +21,8 @@ let basketballTeam = {
 */
 
 // write code here
-
+let topPlayers = basketballTeam.topPlayers;
+let alphabeticalOrder = basketballTeam.topPlayers.sort();
```

Comment:
You need to use console.log() to output each of these - you could use forEach or something similar.